src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -3603,22 +3603,88 @@ protected override bool ProviderSupportsShouldProcess
 
         #region Command code
 
+        private Collection<PathInfo> GetResolvedPaths(string path)
+        {
+            Collection<PathInfo> results = new Collection<PathInfo>();
```

Comment:
It is extra allocation. I think we should remove this and check null in calling code.

Reply:
Updated test.